README.md

```diff
@@ -18,11 +18,28 @@ R = np.array([
     [0, 1, 5, 4],
 ])
 
+
+#because there are many dot products to calculate during the procedure, GPU can be used for this task

```

Comment:
I think this part can be simplified to as follows, because this is not an actual script that can be executed, but just a sample of how to use the module:

```
gpu = False   # Set to True to use GPU in training and prediction
```

Reply:
Yes you are right it is more clear like this, and also according to its significance it is better in the end of the arguments, as you proposed.